### PR TITLE
Add class methods for inspecting verify fail (0x63cX) status words

### PR DESCRIPTION
**Pull request by emlun**
An alternative approach would be to add similar methods to APDUError instead.

### CODE REVIEW

**Comment by dagheyman** [comment]
I'd suggest making `PivController._parse_tries_left` a public method instead.

**Comment by emlun** [comment]
Agreed offline to move `PivController._parse_tries_left` into `SW` instead.

**Comment by dainnilsson** [comment]
Why are these classmethods that take an instance as an argument, instead of properties?

**Comment by emlun** [comment]
Because the enumeration doesn't have constans for all the possible `0x63cX` values:

- `VERIFY_FAIL_RETRY_0 = 0x63c0`
- `VERIFY_FAIL_RETRY_1 = 0x63c1`
- `VERIFY_FAIL_RETRY_2 = 0x63c2`
- `VERIFY_FAIL_RETRY_3 = 0x63c3`
- `COUNTER_VALUE_4 = 0x63c4`
- ... 
- `COUNTER_VALUE_15 = 0x63cf`

and without that it's not possible to have "instances" of the enumeration on which to invoke the properties. We could of course add those enumeration constants, but then `SW(sw).is_verify_fail` would _still_ crash if given a `sw` that doesn't match any of the constants.

**Comment by dainnilsson** [comment]
Ah. In that case, `@staticmethod`?

**Comment by emlun** [comment]
Ah, I didn't know there's that option. Sure!

File: ykman/piv.py [hunk 1]
```diff
@@ -256,6 +256,25 @@ class SW(IntEnum):
     AUTHENTICATION_BLOCKED = 0x6983
     INCORRECT_PARAMETERS = 0x6a80
 
+    @classmethod
+    def is_verify_fail(cls, sw):
+        return 0x63c0 <= sw <= 0x63c3
```
Comment (dainnilsson) [inline ykman/piv.py]: It's not quite clear to me why we're just considering values 0-3 as valid here, but the `tries_left` function handles a much larger span.

Comment (emlun) [reply ykman/piv.py]: Yeah, I agree. The APDU list says

High byte | Low byte | Severity | Explanation
-- | -- | -- | --
63 | C0 | W | Verify fail, no try left.
63 | C1 | W | Verify fail, 1 try left.
63 | C2 | W | Verify fail, 2 tries left.
63 | C3 | W | Verify fail, 3 tries left.
63 | CX | W | The counter has reached the value 'x' (0 = x = 15) (command dependent).

so maybe we should expand `is_verify_fail`? I'm not sure which is more correct.

In actuality, the reason for the discrepancy is that `tries_left` was moved here from `PivController._parse_tries_left` and `is_verify_fail` was newly written before we decided to move `_parse_tries_left`.

Comment (emlun) [reply ykman/piv.py]: Bump

Comment (dagheyman) [reply ykman/piv.py]: I think 63CX should be seen as a "Verify fail" as well, no?

Comment (dainnilsson) [reply ykman/piv.py]: Agreed

Comment (emlun) [reply ykman/piv.py]: All right, will do!

**Pull request merged**